daisy_workflows/image_build/centos/centos6.wf.json

```diff
@@ -4,20 +4,17 @@
     "image-dest": "",
     "installer-iso": ""
   },
-  "Sources": {
-    "installer.iso": "${installer-iso}"
-  },
   "Steps": {
     "build-centos6": {
       "Timeout": "60m",
       "SubWorkflow": {
         "Path": "./centos.wf.json",
         "Vars": {
           "image-dest": "${image-dest}",
-          "installer-iso": "${SOURCESPATH}/installer.iso",
+          "installer-iso": "${installer-iso}",
           "license-id": "1000206",
           "release": "centos6",
-          "wf-name": "build-centos6"
+          "wf-name": "centos6"
```

Comment:
I don't like how long the resource names were getting: installerprep-build-rhel6-abcde vs installerprep-rhel6-abcde
Also, when the build VM was running, it was build-build-rhel6-abcde.